src/App.js

```diff
@@ -1,27 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] It would be better if the App is class-based. Kindly fix this minor issue.

```suggestion
class App extends Component {
  return (
    <div className="App">
```

Reply:
Done

Reply:
This issue is well resolved! ✔️